src/librustc/dep_graph/dep_node.rs

```diff
@@ -93,6 +94,7 @@ pub enum DepNode<D: Clone + Debug> {
     ItemSignature(D),
     ItemVarianceConstraints(D),
     ItemVariances(D),
+    IsConstFn(D),
```

Comment:
note that this whole mechanism recently changed, so you'll have to rebase it-- in the new version, you would just write `DefId` here